.env.default

```diff
@@ -25,3 +25,4 @@ GITHUB_SITE_TITLE=GitHub
 DEVISE_MODULES='[database_authenticatable,recoverable,rememberable,trackable,validatable,omniauthable]'
 GOOGLE_AUTHENTICATION=true
 GOOGLE_SITE_TITLE=Google
+ERRBIT_NOTICE_DEPRECATION_DAYS=7
```

Comment:
And once the name and default is settled, this should be documented in https://github.com/errbit/errbit/blob/master/docs/configuration.md